main.css

```diff
@@ -0,0 +1,50 @@
+@import url('https://fonts.googleapis.com/css?family=Merriweather:900|Roboto');
+
+* {
+    width: 625px;
+    color: white;
+}
+
+span {
+    border: 1px rgba(211, 211, 211, 0.2);
+    background-color: rgba(211, 211, 211, 0.2);
+    color: white;
+    border-radius: 15px;
+    margin: 10px 6px;
+    padding: 6px 10px;
+    display: inline-block;
+    width: auto;
+}
+
+div {
+    padding: 30px 100px 15px 25px;
+    margin-bottom: 25px;
```

Comment:
The task said not to use padding or margin to create spacing between the cards. This is to turn your attention to a commonly overlooked part of the box model - border! So instead of using margin, you can set the same value to border and then just make it invisible, aka transparent.